tests/extract_test.rs

```diff
@@ -125,7 +125,7 @@ fn extract_empty_test() {
         .with(Modify::new(Full).with(Alignment::left()))
         .with(Modify::new(Full).with(Indent::new(3, 1, 0, 0)))
         .with(Modify::new(Full).with(Format(|s| format!("[{}]", s))))
-        .with(Extract::new(1..1, 1..1))
+        .with(Extract::segment(1..1, 1..1))
         .to_string();
 
     assert_eq!(table, "");
```

Comment:
Maybe to add 2 small tests for `Extract::columns` and `Extract::rows`

Reply:
`tabled>tests>extract.rs` has been extended by 6 additional tests for rows and columns